src/components/status.jsx

```diff
@@ -693,9 +693,13 @@ function Status({
     x: 0,
     y: 0,
   });
-  const bindLongPress = useLongPress(
+  const bindLongPressContext = useLongPress(
     (e) => {
       const { clientX, clientY } = e.touches?.[0] || e;
+      // link detection copied from onContextMenu because here it works
+      const link = e.target.closest('a');
+      if (link && /^https?:\/\//.test(link.getAttribute('href'))) return;
```

Comment:
The above two lines work to prevent the context menu when long-pressing a link in Chrome, allowing the default menu. It's the same from onContextMenu, but here it gets triggered correctly. I suppose the filterEvents hook could also work, but this was conveniently here already.